src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/KontantstøtteVedtakAvslagMetadata.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.Fagsystem
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.dokarkiv.Dokumenttype
+import org.springframework.stereotype.Component
+
+@Component
+object KontantstøtteVedtakAvslagMetadata : Dokumentmetadata {
+
+    override val journalpostType: JournalpostType = JournalpostType.UTGAAENDE
+    override val fagsakSystem: Fagsystem = Fagsystem.KONT
+    override val tema: Tema = Tema.KON
+    override val behandlingstema: Behandlingstema = Behandlingstema.Kontantstøtte
+    override val kanal: String? = null
+    override val dokumenttype: Dokumenttype = Dokumenttype.KONTANTSTØTTE_VEDTAK_AVSLAG
+    override val tittel: String = "Vedtak om avslag - kontantstøtte"
+    override val brevkode: String = "KS01"
```

Comment:
https://confluence.adeo.no/display/TITR/Brevkoder+-+Infotrygd
riktig at man skal gjenbruke brevkoden?

Reply:
Litt usikker på om jeg skjønner hva du mener her 🤔 😛

Reply:
Jeg antar at KS01 er et infotrygdbrev. Se liste over infotrygdbrev for KS fra confluence. Så spørsmålet er om man skal gjenbruke brevkoden fra infotrygdbrevet, eller om man skal lage egen som man har gjort ved de andre brevene. Tilsvarende ved KS04. 

Og hvis det er OK, skulle man gjenbrukt flere brevkoder?

Reply:
Hmm.. godt poeng. Ser på tilsvarende metadata-filer for barnetrygd og der har de brukt `BAA1` for både vedtak, innvilgelse og avslag. Slik jeg oppfatter brevkoder generelt i denne løsningen så har det ikke så mye å si hva brevkoden er på utgående dokumenter så er kanskje smartest å lage egne og styre bort bra infotrygd kodene.

Reply:
Ta en sjekk med eivind om det når han er tilbake. Jeg godkjenner. Kan jo alltids endres hvis det var feil.

Reply:
Evt Birgitte/Anna